docs/how-to/generate-blockchain.md

```diff
@@ -0,0 +1,35 @@
+# How to Generate Blockchain
+
+It is possible situation when blockchain is not advanced for more than `k` slots.
+In this case we should generate it. Use `cardano-block-gen` command for it.
+
+## Basic Command
+
+The simplest example of the command:
+
+```
+$ stack exec -- cardano-block-gen                \
+    --system-start 0                             \
+    --configuration-file node/configuration.yaml \
+    --configuration-key mainnet_full             \
+    --blocks 1000                                \
+    --generated-db new-db
+```
+
+where:
+
+*  `1000` is a length of generated blockchain (number of blocks),
+*  `new-db` is a path to generated database.
+
+## Additional Parameters
+
+If the database already exists, we can use `--append` option. In this case new
+database won't be created and records will be appended to existing database.
```

Comment:
I don't think DevOps will ever want not to use `--append`. I would put it into command above.